### DISCUSSION TITLE
openssl version

### DISCUSSION THREAD

**Discussion by Minilo-ng** [prisma/prisma#12771] [Q&A]
I wonder why OpenSSL did not support the version 0.9.x, is it in tech or just ignored?

**Comment by nurul3101** [comment]
Hey @Minilo-ng 👋 
Here's the list of operating systems that we support and their [binaryTargets](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#binarytargets-options) value.

OpenSSL `0.9.x` has been officially out of support and is not receiving updates since 2016 so there's no dire need to support it, you can use `1.0.x` which is supported by Prisma.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
